Mathlib/Analysis/NormedSpace/Alternating/Basic.lean

```diff
@@ -0,0 +1,601 @@
+/-
+Copyright (c) 2020 Sébastien Gouëzel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sébastien Gouëzel, Yury Kudryashov, Heather Macbeth, Patrick Massot
+-/
+import Mathlib.Topology.Algebra.Module.Alternating.Topology
+import Mathlib.Analysis.NormedSpace.Multilinear.Basic
+
+/-!
+# Operator norm on the space of continuous alternating maps
+
+In this file we show that continuous alternating maps
+from a seminormed space to a (semi)normed space form a (semi)normed space.
+We also prove basic facts about this norm
+and define bundled versions of some operations on continuous alternating maps.
+
+Most proofs just invoke the corresponding fact about continuous multilinear maps.
+-/
+
+noncomputable section
+
+open scoped BigOperators NNReal
+open Finset Metric
+
+/-!
+### Type variables
+
+We use the following type variables in this file:
+
+* `𝕜` : a nontrivially normed field;
+* `ι`: a finite index type with decidable equality;
+* `E`, `F`, `G`: (semi)normed vector spaces over `𝕜`.
+-/
+
+/-- Applying a continuous alternating map to a vector is continuous in both coordinates. -/
```

Comment:
perhaps say "in both the map and the vector"?

Reply:
Added an explanation.